src/sentry/eventstore/snuba/backend.py

```diff
@@ -1,10 +1,77 @@
 from __future__ import absolute_import
 
+from datetime import datetime
+import six
 
+from sentry.utils import snuba
 from sentry.eventstore.base import EventStorage
 
 
 class SnubaEventStorage(EventStorage):
     """
     Eventstore backend backed by Snuba
     """
+
+    def get_next_event_id(self, event, conditions=None, filter_keys=None):
```

Comment:
Looking at the next iteration of events/discover, I think there's a pretty high chance that we'll want to find a next event in a given set of projects, rather than only searching for the next event in in the same project as the existing event. I wanted to avoid the single project restriction.

Reply:
I would agree. The pagination in discover2 should be able to traverse the results of any query, which could cross project lines depending on the query.

Reply:
ok